cypress/cypress/apis/clusterSet.js

```diff
@@ -22,7 +22,7 @@ export const getClusterSet = (clusterSet) => {
         method: "GET",
         url:
             constants.apiUrl +
-            constants.ocm_cluster_api_v1beta2_path +
+            constants.ocm_cluster_api_v1beta1_path +
```

Comment:
This change already merged within the branch.
Your patch should not change it again.